pyalex/api.py

```diff
@@ -18,7 +19,14 @@ def __setattr__(self, key, value):
         return super().__setitem__(key, value)
 
 
-config = AlexConfig(email=None, api_key=None, openalex_url="https://api.openalex.org")
+config = AlexConfig(
+    email=None,
+    api_key=None, 
+    openalex_url="https://api.openalex.org",
+    max_retries=0,
+    retry_backoff_factor=0.1,
+    retry_http_codes=[500, 503],
```

Comment:
Shall we add 429? In case you send more than 10 requests a second.

Reply:
Yes, we could also enable by default the retry?

Reply:
Ideally, users respect the rate limit themselves. I prefer to add 429 and stick max_retries to 0.